gamedb.yaml

```diff
@@ -4861,6 +4867,13 @@
   id: 218620
   compat_tool: proton-stable
 
+# Game doesn't start with standard proton 9.0-3
+- name: "Persona 5 Strikers"
+  platform: steam
+  id: 1382330
+  compat_tool: GE-Proton9-13
```

Comment:
Should we update to 9-20 now that it is available?

Reply:
I can only  select a GE version up to 9-13. 
9-20 isn't available in the compatibility tool selection.

Reply:
I added 9-20 a while ago. It should be there. Trying restarting the computer.

Reply:
I wasnÄt on latest release. 
After the upgrade, 9-20 was available.
But now none of the 9 versions work, only an 8, but without working controllers.
going through ProtonDB, this game seems not really reliable stable on a specific GE release.
Hence I decided to remove the entry for Persona 5.

Hence only Casette Beast is part of this PR.